src/javascript/JContent/ContentRoute/ContentLayout/ContentTable/reactTable/components/cells/CellStatus.jsx

```diff
@@ -35,7 +35,7 @@ export const CellStatus = ({cell, column, row}) => {
                     <Chip className={classes.statusCellItem} icon={<Build fontSize="small"/>} color="warning"/>
                 </Tooltip>}
             {node.lockOwner !== null &&
-            <Tooltip title={t('jcontent:label.contentManager.locked')}><Chip className={classes.statusCellItem} icon={<Lock fontSize="small"/>} color={isMarkedForDeletion(node) ? 'danger' : 'warning'}/></Tooltip>}
+            <Tooltip title={t('jcontent:label.contentManager.locked')}><Chip className={classes.statusCellItem} icon={<Lock fontSize="small"/>} color={isMarkedForDeletion(node) ? 'danger' : 'default'}/></Tooltip>}
```

Comment:
Not for right now I guess but maybe this can be replaced in future also with LockInfoChip